_posts/2024-06-30-cryostat-release.md

```diff
@@ -0,0 +1,81 @@
+5. **Reverse proxy architecture**: Cryostat 3.0 deployed by the Cryostat Operator or Helm Chart includes a reverse proxy ([`openshift-oauth-proxy`](https://github.com/openshift/oauth-proxy/) or [`oauth2_proxy`](https://github.com/oauth2-proxy/oauth2-proxy)) in the Pod. Only this proxy is exposed to cluster traffic via a Service. This means that all API requests to Cryostat, as well as all visitors to the [Cryostat Web UI](https://github.com/cryostatio/cryostat-web) or [Grafana dashboard](https://github.com/cryostatio/cryostat-grafana-dashboard) flow through this proxy. The proxy handles user sessions to control access to the application, providing unified access control and user sessions for both the Cryostat Web UI and Grafana dashboard. Both of these UIs are accessible via the same Ingress or Route and present the same TLS certificate. When deployed on [OpenShift](https://developers.redhat.com/products/openshift/overview), the auth proxy will use its Namespace (the Cryostat installation Namespace) to perform Role-Based Access Control checks for user authentication and authorization by integrating with the OpenShift cluster SSO provider, very similar to what previous Cryostat releases directly implemented. Additionally, the auth proxy can be optionally configured with an `htpasswd` file to enable Basic authentication. On OpenShift this allows for defining additional user accounts that may access the Cryostat application beyond those with OpenShift SSO RBAC access, whereas on other Kubernetes it is the primary authentication mechanism.
+
+#### Operator
+
+Specifically for the Cryostat Operator there are a few items of note:
+1. **Cluster-wide installation** only: the Cryostat Operator can now only be installed cluster-wide, rather than into a subset of cluster namespaces. Cluster-wide installation is the preferred mode for the Operator Lifecycle Manager and per-namespace installations are a deprecated feature, which our Operator has now dropped.
+2. **ClusterCryostat removal**: the `v1beta1/Cryostat` and `v1beta1/ClusterCryostat` custom resources are now unified in the `v1beta2/Cryostat` CR. When creating this CR you specify an installation namespace and an optional list of target namespaces. The installation namespace is the location where the Operator will create the Cryostat Deployment, and on OpenShift this namespace will also be used by the auth proxy for RBAC to control which users have access to the Cryostat application. The list of target namespaces is used to control which namespaces the Cryostat instance should monitor for new target applications to appear within, the same as before. Taken together with point 1 above it is now simpler to install the Operator and configure your Cryostat instance(s): simply install the Operator cluster-wide, then create one or more Cryostat CRs to correspond to namespaces or groups of namespaces that contain your applications.
```

Comment:
> When creating this CR you specify an installation namespace and an optional list of target namespaces.

Do you think the users might be mistaking installation namespace as a CR spec rather than the CR's  `.metadata.namespace`?